src/clj/puppetlabs/services/jruby_pool_manager/impl/jruby_internal.clj

```diff
@@ -2,16 +2,20 @@
   (:require [schema.core :as schema]
             [puppetlabs.services.jruby-pool-manager.jruby-schemas :as jruby-schemas]
             [clojure.tools.logging :as log]
-            [puppetlabs.i18n.core :as i18n])
+            [puppetlabs.i18n.core :as i18n]
+            [clojure.java.io :as io]
```

Comment:
nitpick: I try to keep requires in some sort of order (on previous teams we agreed on roughly alphabetical, what works best here tends to depend on whether editors manage this for you)

Reply:
i can rearrange that.